### PR TITLE
First draft of panels CFP

### PR DESCRIPTION
**Pull request by jalevine**
I've added a template for this.  We may still make minor changes to the language of CFP, but this version could also go live

### CODE REVIEW

File: content/info/call-participation/panels.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+We note that submitting similar topics to multiple venues (e.g., panels and workshops) is likely to result in at most one accepted proposal so that a broad range of topics can be presented at the conference.
+
+Submissions should be submitted via [PCS](http://new.precisionconference.com/vgtc/). Please note the [formatting guidelines](http://junctionpublishing.org/vgtc/Tasks/camera.html) when preparing your submission. 
+
+Feel free to address any questions concerning submission, format, and content to [panels@ieeevis.org](mailto:panels@ieeevis.org).
+
+## Past Panels
+Panels will be recorded and posted to Vimeo for archival viewing. Panels from  2018 can be found at [http://ieeevis.org/year/2018/info/panels](http://ieeevis.org/year/2017/info/panels) and from 2019 can be found at [http://ieeevis.org/year/2018/info/panels](http://ieeevis.org/year/2019/info/panels)
```
Comment (mbehrisch) [inline content/info/call-participation/panels.md]: Links go the wrong year

Comment (jalevine) [reply content/info/call-participation/panels.md]: Should be fixed with my last commit

**Comment by yelper** [comment]
Merged!  You should see this page on staging [once it builds](https://github.com/ieee-vgtc/ieeevis.org/runs/430386432) in ~5m: http://staging.ieeevis.org/year/2020/info/call-participation/panels

**Comment by jalevine** [comment]
Hmm.  It seems the table at the end of this did not render correctly.  The weird part is that table is written directly in html (and recycled from past years), but it looks like it's rendering almost like the markdown specification for a table.

Thoughts on how to fix this?

**Pull request merged**